libavfilter/dnn_filter_common.h

```diff
@@ -51,6 +53,7 @@ int ff_dnn_init(DnnContext *ctx, DNNFunctionType func_type, AVFilterContext *fil
 int ff_dnn_set_frame_proc(DnnContext *ctx, FramePrePostProc pre_proc, FramePrePostProc post_proc);
 int ff_dnn_set_detect_post_proc(DnnContext *ctx, DetectPostProc post_proc);
 int ff_dnn_set_classify_post_proc(DnnContext *ctx, ClassifyPostProc post_proc);
+char **ff_dnn_parse_values(DnnContext *ctx, const char *output_exp, const char *val_sep, int *nb_output);
```

Comment:
what does this function do?  We need a meaningful function name.

Reply:
To parse multiple outputs, with '&' to seperate. Maybe parse_output_name_expr better?
While, i thought may be there would be other condition, maybe this func would be called to parse multiple names/expressions with some certain seperation symbol. So i choosed a more common func name.

Reply:
we still need a meaningful name even it is common, for example, separate_expressions or something else.

Reply:
Yes, i get it